### ISSUE TITLE
variable name typo in `dice_loss` error message 

### ISSUE THREAD

**Issue by NickleDave** [kornia/kornia#982]
## 🐛 Bug

Hi, thank you for developing `kornia`.  
This is not a bug per se, more like a typo.  

Just noticed one of the variable names is wrong in an error message in the `dice_loss` function
https://github.com/kornia/kornia/blob/5392651d0bc268da577fa0a49aa50f957289c7dd/kornia/losses/dice.py#L61

```
    if not input.shape[-2:] == target.shape[-2:]:
        raise ValueError("input and target shapes must be the same. Got: {} and {}"
                         .format(input.shape, input.shape))
```

I think last line should instead be
```
                         .format(input.shape, target.shape))
```

Happy to submit a PR if you want or just let you all handle it, minor thing but thought you'd want a heads up

**Comment by shijianjian** [comment]
Great. Please go for a PR.

**Comment by shijianjian** [comment]
Closed by #983.
